app/controllers/activity_types_controller.rb

```diff
@@ -23,16 +23,23 @@ def show
       if show_data_enabled_activity_type?(@activity_type, current_user_school)
         @activity_type_content = @activity_type_content.body.to_html.html_safe
       end
-      @can_be_completed = can_be_completed(@activity_type, current_user_school)
     else
       @activity_type_content = @activity_type.description
     end
+    @can_be_completed_for_schools = can_be_completed_for_schools(@activity_type, current_user.schools) if current_user
+  end
+
+  def for_school
+    school = School.find(params[:school_id])
+    redirect_to new_school_activity_path(school, activity_type_id: @activity_type.id)
   end
 
   private
 
-  def can_be_completed(activity_type, school)
-    ActivityTypeFilter.new(school: school).activity_types.include?(activity_type)
+  def can_be_completed_for_schools(activity_type, schools)
```

Comment:
We could change this so that we avoid the performance overhead for admins by letting them record any activity for any school. I think that's reasonable as its a super user privilege. But for group admins we'd want to enforce the usual rules.